canonical-web-and-design/vanilla-framework
# Add import snippet to components documentation

## Done

- Added a section in the context menu called 'Import' to components
- Provides import snippet to paste into your project
- Links to Customising Vanilla page for further information

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/
- Check 'Import' section applied to specific components that require single import
  - accordion
  - article-pagination
  - breadcrumbs
  - buttons
  - card
  - code
  - contextual-menu
  - forms
  - heading-icon
  - headings
  - icons
  - image
  - inline-images
  - label
  - links
  - list-tree
  - lists
  - matrix
  - media-object
  - modal
  - muted-heading
  - navigation
  - notifications
  - pagination
  - pull-quotes
  - search-box
  - slider
  - strip
  - switch
  - tables
  - table-of-contents
  - tabs
  - tooltips

## Details

Fixes https://github.com/canonical-web-and-design/vanilla-framework/issues/2549

## Screenshots

<img width="929" alt="Screenshot 2019-10-30 at 10 01 24" src="https://user-images.githubusercontent.com/17748020/67848285-43572d00-fafc-11e9-8116-027b3199f3e3.png">
